File: euFerro-cs50-problems-2022-x-mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    int increaseHash = 1;
    int decreaseSpace = 1;
    int counter;

    //prompt user to type the height
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    //Creates Pyramid

    //EACH ROW
    for (int i = 0; i < height; i++)
    {
        counter = 0;
        //SPACES BEFORE EACH ROW
        for (int s = 0; s < height - decreaseSpace; s++)
        {
            printf(" ");
        }
        //EACH COLUMN
        for (int j = 0; j < 2; j++)
        {
            //EACH SIDE
            for (int b = 0; b < increaseHash; b++)
            {
                printf("#");
            }
            //ONE SPACE IN MIDDLE
            for (int q = 0; q <= counter; q++)
            {
                printf("  ");
            }
            counter--;
        }
        increaseHash++;
        decreaseSpace++;
        printf("\n");
    }
}

File: euFerro-cs50-problems-2022-x-sentimental-mario-more/mario.py
from cs50 import get_int

# Generates a pyramid from height passed by user


def main():
    print_pyramid()

# Print pyramid


def print_pyramid():
    while True:
        n = get_int("Height: ")
        if n >= 1 and n <= 8:
            break
    i = 1
    spc = n
    for n in range(n):
        print(" "*(spc-i) + ("#")*i + "  " + ("#")*i)
        i += 1
    return


# Calls main method
main()